### PR TITLE
[CUR-940] Switch curriculum visualiser to newer MV

### PR DESCRIPTION
**Pull request by BTitterington**
## Description

Music year: 2005

- List of changes

This change consolidates the curriculum visualiser and docs to one MV rather than two.

I've run this through the curriculum visual testing tool and can see no visual changes to units.  

## How to test

1. Go to https://deploy-preview-2853--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Check the units are correct

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/node-lib/curriculum-api-2023/queries/curriculumUnits/curriculumUnits.gql
Comment (BTitterington) [inline src/node-lib/curriculum-api-2023/queries/curriculumUnits/curriculumUnits.gql:1]: I opted to change the existing query in order to get this done quickly.

**Review by sonalisin — APPROVED** [review]

**Review by orangemug — APPROVED** [review]
LGTM

**Review by rightsaidjames — APPROVED** [review]
Looks good - no regressions found compared to production, and all of the complexities in KS4 are working as they should.

Tested in the preview branch for cycle 1, and locally with cycle 2 enabled for the new lots.

**Pull request merged**